desktop/apps/artwork/components/current_auctions/test/current_auctions.js

```diff
@@ -0,0 +1,103 @@
+import 'moment-timezone'
+import CurrentAuctions from '../current_auctions'
+import React from 'react'
+import { expect } from 'chai'
```

Comment:
Chai isn't a framework, its an assertion library (typically used with Mocha), where `mocha` could best be compared to `jest` or `jasmine`. I was following the instructions for `enzyme` [here](http://airbnb.io/enzyme/docs/guides/mocha.html) but after your comment I just tried an assertion without it and its not necessary. Will remove!